includes/opdat_4.2.php

```
@@ -137,16 +865,43 @@ function opdat_4_2($majorNo, $subNo, $fixNo){
 		include("../includes/online.php");
 		if ($db!=$sqdb){
-
+			$qtxt="UPDATE grupper set box1='$nextver' where art = 'VE'";
+			db_modify($qtxt,__FILE__ . " linje " . __LINE__);
+		}
+		include("../includes/connect.php");
+		$qtxt="UPDATE regnskab set version = '$nextver' where db = '$db'";
+		db_modify($qtxt,__FILE__ . " linje " . __LINE__);
+	}
+	$nextver='4.3.0';
+	if ($SubNo < '1'){
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$SubNo'. (undefined)

(UnusedLocalVariable)